runtime/onert/core/src/compiler/train/TensorRegistries.h

```diff
@@ -20,6 +20,8 @@
 #include "../../backend/builtin/Config.h"
 #include "../../backend/builtin/train/TensorRegistry.h"
 
+#include <backend/train/ITensorRegistry.h>
+#include <backend/train/ITrainableTensor.h>
```

Comment:
I think this _#include_ would be better to be removed as the same inclusion is removed by https://github.com/Samsung/ONE/pull/12269#discussion_r1423614066.